internal/rest/avi_obj_vsvip.go

```diff
@@ -487,8 +502,13 @@ func (rest *RestOperations) AviVsVipCacheAdd(rest_op *utils.RestOp, vsKey avicac
 						utils.AviLog.Infof("key: %s, msg: invalid type for vip in vsvip: %s", key, name)
 						continue
 					}
+					ipType := "V4_ONLY"
```

Comment:
we can use already defined constant here.

Reply:
fixed